modules/logging-upgrading-clo.adoc

```diff
@@ -22,12 +22,39 @@ To update the {clo} to a new major release version, you must modify the update c
 
 . Click the *Red Hat OpenShift Logging* Operator.
 
-. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.y*, depending on your current update channel.
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.<y>*, depending on your current update channel.
 
-. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.y*, and click *Save*. Note the `cluster-logging.v5.y.z` version.
+. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.<y>*, and click *Save*. Note the `cluster-logging.v5.<y>.<z>` version.
 
-.Verification
-
-. Wait for a few seconds, then click *Operators* -> *Installed Operators*. Verify that the {clo} version matches the latest `cluster-logging.v5.y.z` version.
+. Wait for a few seconds, and then go to *Operators* -> *Installed Operators* to verify that the {clo} version matches the latest `cluster-logging.v5.<y>.<z>` version.
 
 . On the *Operators* -> *Installed Operators* page, wait for the *Status* field to report *Succeeded*.
+
+[IMPORTANT]
+====
+
+If you are using the {loki-op} with the {clo}, the {clo} 5.9 or later requires the `v13` schema version in the `LokiStack` custom resource.
+
+The correct way to add the `v13` schema version is the following:
+
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+# ...
+spec:
+# ...
+  storage:
+    schemas:
+    # ...
+      version: v12 # <1>
+    - effectiveDate: "<yyyy>-<mm>-<future_dd>" # <2>
+      version: v13
+# ...
+----
+<1> Do not delete. Keep the `v12` configuration to avoid data loss.
```

Comment:
- Customers might also still just have `v11`.
- They do not need to go though v12 to reach v13.
- Data is only written in a certain schema and never modified/updated to a newer one. That's why the new configuration needs to be in the future. And also why the schema configuration needs to continue to contain the old versions, so that this data stays readable.

Reply:
I've updated this line to as follows:
```
<1> Do not delete. Data persists in its original schema version. Keep the previous schema versions to avoid data loss.
```